### ISSUE TITLE
AQLM Quantization

### ISSUE THREAD

**Issue by CamiloMM** [oobabooga/text-generation-webui#5465]
**Description**

AQLM ([GitHub](https://github.com/Vahe1994/AQLM), [Paper](https://arxiv.org/pdf/2401.06118.pdf), [Reddit discussion](https://www.reddit.com/r/LocalLLaMA/comments/1al58xw/yet_another_state_of_the_art_in_llm_quantization/)) is a novel quantization method that  focuses on 2-2.5 bit and claims to surpass [QuiP#](https://cornell-relaxml.github.io/quip-sharp/) and allows for a 70b to run on a 3090 with surprisingly good PPL (allegedly), and even 3-bit GPTQ

**Additional Context**

According to my high-accuracy crystal ball which I bought from The Onion a decade ago, TheBloke will ignore this and continue to release half a dozen quants of GPTQ per model until late 2026, no matter what else dethrones it.

**Comment by oobabooga** [comment]
https://github.com/oobabooga/text-generation-webui/pull/5466

Performance is pretty bad, but it may become better after https://github.com/huggingface/transformers/pull/27931.

**Comment by Tedy50** [comment]
I think at this time exl2 format is most attractive and may become most popular. 

GPTQ  is currently unavoidable because that's the only format which supports training.
but comparing performance with transformers is pointless.

**Comment by CamiloMM** [comment]
> I think at this time exl2 format is most attractive and may become most popular.

Dunno about that man, have you checked the new [IQ2/IQ3_XSS quants?](https://github.com/ggerganov/llama.cpp/pull/5196)

**Comment by Tedy50** [comment]
> Though for now I use exl2 because it's just so fast.)
yes that's exactly why it is most attractive as it is running on exlama engine which is in order of magnitude faster than anything else and allows huge context sizes.  At this time I just don't see any usable alternatives.

considering performance, it is hard to tell how well those new quantization methods perform.  Everybody claims huge success but it doesn't seem to be that good at all. 
to me the main issue with quantization seems to be that models become very unstable at anything below 3 bit

**Comment by dsent** [comment]
It seems that AQLM is supported by now, but there's no way to use it on Windows, because of aqlm[gpu] dependency which requires triton, triton not being available on Windows. @oobabooga, can you confirm?

**Comment by DmitryVN** [comment]
AQLM?
